chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -0,0 +1,258 @@
+# ChainIndexer Documentation for RPC Providers
+
+# Table of Contents
+
+- [ChainIndexer Documentation for RPC Providers](#chainindexer-documentation-for-rpc-providers)
+  - [Introduction](#introduction)
+  - [ChainIndexer Config](#chainindexer-config)
+    - [Enablement](#enablement)
+    - [Garbage Collection](#garbage-collection)
+      - [Recommendations](#recommendations)
+    - [Removed Options](#removed-options)
+  - [Migration Guide](#migration-guide)
+    - [Backfilling](#backfilling)
+      - [The `ChainValidateIndex` JSON RPC API](#the-chainvalidateindex-json-rpc-api)
+      - [`lotus-shed chainindex validate-backfill` CLI tool](#lotus-shed-chainindex-validate-backfill-cli-tool)
+        - [Usage](#usage)
+        - [Parameters](#parameters)
+        - [Error conditions](#error-conditions)
+        - [Logging](#logging)
+        - [Example usage](#example-usage)
+  - [Rollback](#rollback)
```

Comment:
Indent one more level so this is under Migration Guide?  I think that make sense: two main sections: backfilling, rollback.

Reply:
The grammar is strong with this one 🤟

Reply:
Done.